scripts/projects-test/packPackages.ts

```diff
@@ -76,8 +78,12 @@ export async function packProjectPackages(
         );
       }
 
-      await sh(`yarn pack --filename ${filename}`, packagePath);
-      packedPackages[packageName] = filename;
+      // Use `npm pack` because `yarn pack` incorrectly calculates the included files when the
+      // files to include/exclude are specified by .npmignore rather than package.json `files`.
+      // (--quiet outputs only the .tgz filename, not all the included files)
+      const packFile = (await sh(`npm pack --quiet ${packagePath}`, tmpDirectory, true /*pipeOutputToResult*/)).trim();
```

Comment:
This is the part that also affects the northstar tests. The comment explains the reasoning, and the tests still pass, just wanted to raise awareness and make sure we didn't miss any potential issues.